charts/kubeai/values-eks.yaml

```diff
@@ -0,0 +1,29 @@
+resourceProfiles:
+  nvidia-gpu-l4:
+    nodeSelector:
+      # This only works if you use Karpenter.
+      karpenter.k8s.aws/instance-gpu-name: "l4"
+  nvidia-gpu-l40s:
+    nodeSelector:
+      # This only works if you use Karpenter.
+      karpenter.k8s.aws/instance-gpu-name: "l40s"
+  nvidia-gpu-h100:
+    nodeSelector:
+      # H100 have 8 GPU machine shapes only. This doesn't require Karpenter.
+      node.kubernetes.io/instance-type: "p5.48xlarge"
```

Comment:
It would be preferable to avoid specifying an instance-type here if at all possible

Reply:
done, switched to using karpenter nodeSelectors everywhere.